src/sysinfo.rs

```diff
@@ -216,14 +217,22 @@ mod test {
         assert!(s.get_users().len() >= MIN_USERS);
     }
 
+    // FIXME we are still working out the windows API for proper version info
+    // see https://github.com/GuillaumeGomez/sysinfo/issues/415
+    // and https://github.com/GuillaumeGomez/sysinfo/issues/410
+    #[cfg(not(target_os = "windows"))]
```

Comment:
Please don't put the whole function into the `cfg`. You can put a `if` condition for the `get_kernel_version` function using the [`cfg!` macro](file:///home/imperio/rust/rust/build/x86_64-unknown-linux-gnu/doc/std/macro.cfg.html).

Reply:
fixed in 81c7eb0